docs/guides/08-interactivity-events.md

```diff
@@ -0,0 +1,411 @@
+```js
+// Add zoom controls
+const nav = new mapboxgl.NavigationControl();
+map.addControl(nav, 'top-left');
+```
+
+### Map events
+It all begins with the map and sometimes you are interested in listening to some relevant events from to the **map** itself. For example you want to wait for it to load or maybe display the current map's center coordinates. In those cases, you can use a set of events already provided by the `Mapbox GL JS Map`, such as [load](https://www.mapbox.com/mapbox-gl-js/api#map.event:load) and [move](https://www.mapbox.com/mapbox-gl-js/api#map.event:move) respectively, and attach some callback functions to react on them.
+
+Add this pair of listeners to your code to test map events, just after the map initialization:
+```js
+// Wait for the map to render for the first time
+map.on('load', () => {
+    console.log('Map has loaded!');
+});
+// Listen to every move event caused by the user
+const displayCenter = () => {
+    const center = map.getCenter();
+    const longitude = center.lng.toFixed(6);
+    const latitude = center.lat.toFixed(6);
+    const bearing = map.getBearing().toFixed(0);
+    const zoom = map.getZoom().toFixed(2);
+    console.log(`Center: [${longitude}, ${latitude}] - Zoom: ${zoom} - Bearing: ${bearing}º`);
+};
+map.on('move', displayCenter);
+```
+> Check the console after loading your file in the browser, to see the first message: 'Map has loaded'. Then, interact with the map control and see the updated values for center & zoom. You can check Mapbox [Map reference](https://www.mapbox.com/mapbox-gl-js/api/#map) for more information on map events.
+
+The result should look like this:
+<div class="example-map">
+    <iframe
+        id="guides-interactivity-step-1"
+        src="/developers/carto-vl/examples/maps/guides/interactivity/step-1.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+
+### Layer events
+Once you have your basemap, you can start to work with your layers. And all [carto.Layer](/developers/carto-vl/reference/#cartolayer) objects have two events to listen to: `loaded` and `updated`.
+
+The use of the `loaded` event is pretty common, due to in most cases you need to load the data from an external server and that can take some time.
+
+Add this to your code to create a layer as usual, and add it to your map:
+```js
+carto.setDefaultAuth({ user: 'cartovl', apiKey: 'default_public' });
+const source = new carto.source.Dataset('populated_places');
+const viz = new carto.Viz();
+const layer = new carto.Layer('Cities', source, viz);
+layer.addTo(map);
+```
+
+Now add a `loaded` listener to the previous layer:
+```js
+layer.on('loaded', () => {
+    console.log('Cities layer has been loaded!');
+});
+```
+> This event could be useful for example to display some kind of loading animation over your map and then hiding it after the layer has loaded. Notice how the name of the event is `loaded`, not `load`.
```

Comment:
> For example, this event is useful to display a loading animation over your map, hiding it once the layer has loaded. Notice how the name of the event is `loaded`, not `load`.

Reply:
I'm thinking that here we could include a loader example. I'm not sure of how to do so. Maybe just linking to an external example instead of embedding one. What do you think?

Reply:
👍 I'll include a link to an example exemplifying the "loader thing".  As we were removing the loader from common examples, I'll point to `hurricane-harvey.html` (from new landing-page). Pinging here @makella for her to know